.github/workflows/ci.yml

```diff
@@ -8,25 +8,25 @@ on:
       - master
 jobs:
   build:
+    runs-on: ${{ matrix.os }}
+    if: true
     strategy:
       fail-fast: false
       matrix:
         os:
-          - macos-latest
           - ubuntu-latest
-          - windows-latest
+          - macos-latest
+          # - windows-latest
+          - macos-12
+          - macos-13
         ocaml-compiler:
-          - 4.14.x
-        include:
-          - os: ubuntu-latest
-            ocaml-version: 4.08.0
```

Comment:
We probably still want to test it since we have this opam constraint:

https://github.com/geneweb/geneweb/blob/e621bc05cfaa2907c31dcb3ca9e5008699b2ed99/dune-project#L33

Reply:
Of course (again), but since your change in December there is no build on 4.08 at all anymore (see all previous CI in 2024). This is the main goal of those tests, restoring 4.08.x.

Reply:
See https://github.com/geneweb/geneweb/pull/1837.